lib/lambda_ethereum_consensus/state_transition/epoch_processing.ex

```diff
@@ -18,4 +18,25 @@ defmodule LambdaEthereumConsensus.StateTransition.EpochProcessing do
 
     {:ok, state}
   end
+
+  @doc """
+  Process total slashing balances updates during epoch processing
+  """
+  @spec process_slashings_reset(BeaconState.t()) :: {:ok, BeaconState.t()} | {:error, binary()}
+  def process_slashings_reset(
+        %BeaconState{ slashings: slashings } = state
+      ) do
+    next_epoch = Accessors.get_current_epoch(state) + 1
+    slashed_exit_length = ChainSpec.get("EPOCHS_PER_SLASHINGS_VECTOR")
+    slashed_epoch = rem(next_epoch, slashed_exit_length)
+
+    if length(slashings) != slashed_exit_length do
```

Comment:
why this condition? Im not seeing it in the python implementation

Reply:
Since the length of state.slashings should be fixed to [EPOCH_PER_SLASHINGS_VECTOR](https://eth2book.info/capella/part3/config/preset/#epochs_per_slashings_vector), it is to prevent unexpected behaviors and errors.

Reply:
Hmm yeah, I see that `slashings: Vector[Gwei, EPOCHS_PER_SLASHINGS_VECTOR]`.

I don't think this validation is needed though, because if `slashings` length wasn't exactly that size, then the SSZ enconding/decoding/hash_root_tree wouldn't work.

Reply:
Got it! Fixed and committed.